### DISCUSSION TITLE
Abp 7.3 User RefreshSignIn

### DISCUSSION THREAD

**Discussion by burakdmir** [abpframework/abp#17270] [Q&A]
I was using the RefreshSignInAsync method to prompt the user to sign in again, but now I am unable to do so. How can I use it? For example, I am performing an authentication process, and I want to see this change reflected in the CurrentUser.

await _signInManager.RefreshSignInAsync(user);

**Comment by maliming** [comment]
hi

> but now I am unable to do so.

What's the behavior now?

Please share some code and logs.

**Reply by burakdmir** [reply]
Before Running Code

`private readonly SignInManager<IdentityUser> _signInManager;`

`var user = await UserRepository.FindAsync(CurrentUser.GetId());
await _signInManager.RefreshSignInAsync(user);`


but now I am unable to do so.

**Reply by maliming** [reply]
I do not understand this very well. Can you share a simple project?
